src/testdir/test_normal.vim

```diff
@@ -3818,24 +3818,33 @@ func Test_mouse_shape_after_failed_change()
   CheckCanRunGui
 
   let lines =<< trim END
+    vim9script
     set mouseshape+=o:busy
     setlocal nomodifiable
-    let g:mouse_shapes = []
+    var mouse_shapes = []
 
-    func SaveMouseShape(timer)
-      let g:mouse_shapes += [getmouseshape()]
-    endfunc
+    def SaveMouseShape()
+      mouse_shapes += [getmouseshape()]
+    enddef
 
-    func SaveAndQuit(timer)
-      call writefile(g:mouse_shapes, 'Xmouseshapes')
+    def SaveAndQuit()
+      writefile(mouse_shapes, 'Xmouseshapes')
       quit
-    endfunc
+    enddef
```

Comment:
I guess the body of these two functions can just be moved into the lambdas.

Reply:
Yeah that's true.